assets/css/bpmn-js-bpmnlint.css

```diff
@@ -176,22 +199,23 @@
   color: #444;
 }
 
-.bjsl-button svg {
-  margin-top: -1px;
+.bjsl-button .icon {
   margin-right: 10px;
+  width: var(--bjsl-icon-dimensions);
+  height: var(--bjsl-icon-dimensions);
 }

```

Comment:
The `x` seems to not be centered anymore, when it was previously:

![image](https://github.com/bpmn-io/bpmn-js-bpmnlint/assets/21984219/4839a406-9bce-455c-9d3c-341a8b84e5c1)
![image](https://github.com/bpmn-io/bpmn-js-bpmnlint/assets/21984219/e445df08-6eb8-44ca-8484-3253459880ca)

Restricting the height to the child element does the trick

```suggestion
.bjsl-button .icon {
  height: 12px;
  margin-right: 10px;
  width: var(--bjsl-icon-dimensions);
  height: var(--bjsl-icon-dimensions);
}

```

Reply:
Other than that, we are good to go 🚀

Reply:
I spent way to long trying to clean this up and now consider to move back to fixed (12px) icons. So kinda applying your suggestion :see_no_evil:.

Reply:
This should now be cleaned up via https://github.com/bpmn-io/bpmn-js-bpmnlint/pull/48/commits/5686ab38429cf489805704a08fa852964b207a60:

![capture bjcumI_optimized](https://github.com/bpmn-io/bpmn-js-bpmnlint/assets/58601/0870cce2-0afa-4977-b935-ed90d1291dd8)

Further UI alignmens likely to follow in the future.